### DISCUSSION TITLE
📚 GSOC project : out-of-core calculations

### DISCUSSION THREAD

**Discussion by erwanp** [radis/radis#567] [General]
Note regarding GSOC project  Out of Core Claculations https://openastronomy.org/gsoc/gsoc2023/#/projects?project=out-of-core_calculations_to_reduce_memory_usage

Ressources : 

- Polar : https://towardsdatascience.com/7-easy-steps-to-switch-from-pandas-to-lightning-fast-polars-and-never-return-b14c66fc85b9 
- Vaex : https://medium.com/@jovan.veljanoski/8-powerful-vaex-dataframe-features-you-might-have-not-known-about-b2e2d3d3ee9

**Comment by 1someshverma** [comment]
Thanks for it
